README.md

```diff
@@ -1,3 +1,8 @@
+[![PyPI](https://img.shields.io/pypi/v/kappa-maki)](https://pypi.org/project/kappa-maki/)
```

Comment:
I (Informational): add badges for "at a glance" understand on the top of the readme